src/main/java/org/ecommerce/repositories/ProductRepository.java

```diff
@@ -2,6 +2,7 @@
 
 import org.ecommerce.models.Product;
 import org.ecommerce.repositories.impl.CrudOperationsImpl;
+import org.springframework.stereotype.Repository;
```

Comment:
Is this import meant to be used as in `CategoryRepository`?

Reply:
Not really, It was actually an unused import jeje https://github.com/algutierrez-griddynamics/e-commerce/commit/a603af1ca7d824cf189a20073011f7769b8cd100